src/GraphQL/ObjectExtensions.cs

```diff
@@ -123,7 +124,15 @@ public static Type GetInterface(this Type type, string name)
         public static object ConvertValue(object value, Type fieldType)
         {
             if (value == null) return null;
-            if (fieldType == typeof(DateTime) && value is DateTime) return value;
+
+            if (fieldType == typeof(DateTime))
```

Comment:
Instead of making a change here, it should be done in the `DateTimeConverter`.  That is where all of the actual conversion of values are taking place.  After messing with it a bit, I think this is the change that is needed:

```csharp
// var success = DateTime.TryParseExact(trimmedString, "yyyy-MM-dd'T'HH:mm:ss.fffffff'Z'", CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal, out result);
var success = DateTime.TryParse(trimmedString, CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal, out result);
```